nautobot/dcim/tests/test_views.py

```diff
@@ -1136,40 +1136,6 @@ def test_devicetype_export(self):
         self.assertEqual(data[0]["manufacturer"], device_type.manufacturer.name)
         self.assertEqual(data[0]["model"], device_type.model)
 
-    @override_settings(EXEMPT_VIEW_PERMISSIONS=["*"])
-    def test_rack_height_bulk_edit_set_zero(self):
```

Comment:
Feels like this logic should be reimplemented in a Job-based test case rather than removed entirely?